index.html

```diff
@@ -61,11 +92,37 @@ <h2 id="multi-post-title" >Multi-Post Stories</h2>
                                 <li><a href="#"><button class="multi-post-button">Ruby</button></a></li>
                             </ul>
                         </div>
+                        <a href="#"><button class="first-see-project-button-desktop">See Project</button></a>
                     </div>
                     <a href="#"><button class="first-see-project-button">See Project</button></a>
                 </div>
+            </div>
+            <div class="desktop-project-div">
+                <div class="project-1 project-2">
+                    <img class="mobile-backdrop" src="images/Img Placeholder-2.png" alt="printed-magazine">
+                    <!-- <div> -->
+                        <img class="project-img-1 desktop-backdrop" src="images/prof-printing.png" alt="background-image">
+                        <div class="floating-1 floating">
+                            <div>
+                                <h3 class="project-title">Profesional Art Printing Data</h3>
+                            </div>
+                            <div>
+                                <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+                            </div>
+                            <ul class="project-1-buttons-ul">
+                                <li><a href="#"><button class="project-1-buttons">html</button></a></li>
+                                <li><a href="#"><button class="project-1-buttons" >bootstrap</button></a></li>
+                                <li><a href="#"><button class="project-1-buttons">Ruby</button></a></li>
+                            </ul>
+                        </div>
+                        <div class="see-project-div">
+                            <a href="#"><button class="see-project-button-1">See Project</button></a>
+                        </div>
```

Comment:
- [ ] The project cards have 2 states, the default state and on hover state, the default state looks like this 👇 


![image](https://user-images.githubusercontent.com/50721479/208094325-34db94ae-832c-40f5-a570-279a462d02d8.png)

and on hover, it becomes like this 👇 

![image](https://user-images.githubusercontent.com/50721479/208094375-15fdcb75-b5e7-472b-b17d-f23afcebbc7b.png)

Kindly make the necessary changes to satisfy this requirement as requested by the previous reviewer 💯

Reply:
@ha-manel and @emmiiorji Please this was not a required for the project. We were asked to make the buttons to hover not the images. Please let me know the way forward

Reply:
Hi @coderacheal, I get your confusion since in Figma the hover effect is not very visible. Please check the following review statements that clarify the way the cards are designed 👇 

![image](https://user-images.githubusercontent.com/50721479/208115952-360f6e88-40c4-46fd-8540-b5d79162f850.png)
![image](https://user-images.githubusercontent.com/50721479/208115993-85123a97-9b0d-485d-b5b6-673b963cffb8.png)

This means the cards have their description on top of the image with a dark opacity, when the card is hovered the description disappears and instead we see a button that says "See project" 👍

Reply:
@ha-manel please where can I find the extract you just sent me. It is not part of my project description. Can you please direct me to where I can find it.

Reply:
@coderacheal unfortunately, It's not available to you because this is for code reviewers only, but following the Figma design is required even though they don't describe the design in details 😅 As I said before, the design is a bit confusing because you don't see the hovering but I'm sure now it's clear and you can start working on it 😊

Reply:
Please none of my colleagues said they were asked and since I don't see the requirement I cannot implement it.  I have checked the project page multiple times. I am also yet to do my animation project, and I need to work on this with my partner. As I was only required to work on the button effects, that is what I have done. 

Besides, this project comes before we do animations so I am not sure how I am required to do that when we had not yet done animations.

Reply:
@coderacheal you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). Best of luck !

Reply:
@ha-manel Thank you